src/pages/chart/index.tsx

```diff
@@ -37,21 +37,27 @@ function Chart1() {
   const opacity = "0.04";
   const areastyle = { opacity: opacity };
 
+  const legendColor = useColorModeValue("#222", "#EEE");
```

Comment:
i'm trying to figure out how to have the colors switched in the theme, unfortunately i have not got it figured yet, so this seems the best way for now